template/node/Dockerfile

```diff
@@ -12,27 +12,30 @@ WORKDIR /root/
 # Turn down the verbosity to default level.
 ENV NPM_CONFIG_LOGLEVEL warn
 
+RUN mkdir -p /home/app
+
 # Wrapper/boot-strapper
-COPY package.json       .
+COPY package.json       /home/app
+
+WORKDIR /home/app
 RUN npm i
 
 # Function
-COPY index.js           .
-RUN mkdir -p ./root/function
+COPY index.js           /home/app
 
-COPY function/*.json    ./function/
-WORKDIR /root/function
+COPY function/*.json    /home/app/function/
+WORKDIR /home/app/function
 RUN npm i || :
-WORKDIR /root/
-COPY function           function
-WORKDIR /root/function
+WORKDIR /home/app/
+COPY function           ./function
 
-WORKDIR /root/
+RUN addgroup -S app && adduser -S -g app app
```

Comment:
We can add the group at the top - meaning this gets cached and not affected by copying code in.

Reply:
i can make this change @alexellis then we should be all set. should I make the change, squash and resubmit? or can i squash and commit with this same PR?